xhyve/xhyve.go

```diff
@@ -414,7 +414,7 @@ func (d *Driver) PreCreateCheck() error {
 	if err != nil {
 		return fmt.Errorf("Error detecting VBox version: %s", err)
 	}
-	if !strings.HasPrefix(ver, "5") {
+	if strings.HasPrefix(ver, "4") {
```

Comment:
👍

Reply:
So, what you fix means, is:
- If a someone uses a Virtualbox Version lower than 4, there will be no error message
- If at some point we are t Virtualbox 40, the check will fail